migrations/db/migrations/20230224042246_grant_extensions_perms_for_postgres.sql

```diff
@@ -0,0 +1,8 @@
+-- migrate:up
+grant select on all tables in schema extensions to postgres, anon, authenticated, service_role;
+grant execute on all functions in schema extensions to postgres, anon, authenticated, service_role;
+alter default privileges in schema extensions grant select on tables to postgres, anon, authenticated, service_role;
+alter default privileges in schema extensions grant execute on functions to postgres, anon, authenticated, service_role;
```

Comment:
Not sure about the implications of doing this - maybe it's safer to add custom scripts per extension as needed? There may be some extensions like pgsodium where it's not safe to expose certain functions to some of the roles.

Reply:
How about giving postgres grant option? I think minimally platform users should be able to control the roles that can access their enabled extensions. That way they can self serve such use cases in the future.

```sql
grant select on all tables in schema extensions to postgres with grant option;
grant execute on all functions in schema extensions to postgres with grant option;
alter default privileges in schema extensions grant select on tables to postgres with grant option;
alter default privileges in schema extensions grant execute on functions to postgres with grant option;
```

Reply:
seems like postgres already has these grant options on extensions schema.

Reply:
Pgsodium creates its own schema rather than installing to extensions. As a general rule, we can make extensions schema public and if there's something users shouldn't change, they can be installed in their own schema.

Reply:
Yeah, giving `postgres` grant option sounds fine.